### PR TITLE
feat: git resource detector

### PR DESCRIPTION
**Pull request by blumamir**

### CODE REVIEW

**Comment by blumamir** [comment]
.

File: detectors/node/resource-detector-git/src/fecth-git-data.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import child_process from 'child_process';
+import path from 'path';
+import fs from 'fs';
+
+export const executeGitCommand = (gitCommand: string): string => {
+    return child_process
+        .execSync(gitCommand, { stdio: ['ignore', 'pipe', 'pipe'] })
```
Comment (YanivD) [inline detectors/node/resource-detector-git/src/fecth-git-data.ts:7]: Maybe add `timeout` option for execSync? 
@blumamir

**Review by YanivD — APPROVED** [review]
👌

File: detectors/node/resource-detector-git/src/fecth-git-data.ts (hunk 1 above)
Comment (blumamir) [reply detectors/node/resource-detector-git/src/fecth-git-data.ts:7]: good idea 👍

**Pull request merged**